terminal/TerminalClient.cpp

```diff
@@ -139,7 +278,7 @@ int main(int argc, char** argv) {
   FLAGS_logbuflevel = google::GLOG_INFO;
   srand(1);
 
-  Options options = {
+  Options o = {
```

Comment:
Let's revert this back to putting options on the stack now that we fixed the other issue.